.template-lintrc.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
     rules: {
         'block-indentation': 4,
         'no-bare-strings': true,
-        'no-implicit-this': true,
+        'no-implicit-this': { allow: ['hasBlock'] },
```

Comment:
Looks like, instead of `hasBlock`, they [now recommend the `has-block` helper](https://guides.emberjs.com/v3.4.0/components/block-params/#toc_supporting-both-block-and-non-block-component-usage-in-one-template), which doesn't require a lint exception.